ci/testnet-manager.sh

```diff
@@ -196,23 +209,47 @@ sanity() {
   esac
 }
 
+deploy() {
+  declare maybeCreate=$1
+  declare maybeStart=$2
+  declare maybeStop=$3
+  declare maybeDelete=$4
 
-start() {
-  declare maybeDelete=$1
-  if [[ -z $maybeDelete ]]; then
-    echo "--- start $TESTNET"
+  # If $1 is set, we will [re]create the nodes
+  if [[ -z $maybeCreate ]]; then
+    skipCreate=skip
+  else
+    skipCreate=""
+    echo "--- create $TESTNET"
+  fi
+
+  # If $2 is set, we will [re]start the network software on the nodes
```

Comment:
nit: no need for "If $2 is set" ("If $3 is set", etc) as a comment, that's literally what the `if` statement on the next line already says